newIDE/app/src/UI/SemiControlledAutoComplete.js

```diff
@@ -84,6 +84,7 @@ export const autocompleteStyles = {
     padding: 0,
     margin: 0,
   },
+  listbox: { padding: 0, margin: 0 },
```

Comment:
do you need a maxHeight & overflowY: scroll here too?

Reply:
No, I just moved it to this exported variable and it doesn't look like this autocomplete needs it